nuntium/templatetags/nuntium_tags.py

```diff
@@ -35,3 +38,15 @@ def join_with_commas(obj_list):
 @register.assignment_tag(takes_context=True)
 def assignment_url_with_subdomain(context, view, subdomain=UNSET, *args, **kwargs):
     return subdomainsurls(context, view, subdomain, *args, **kwargs)
+
+
+@register.simple_tag(takes_context=True)
+def custom_stylesheet(context):
+    output = ''
+    if hasattr(context['request'], 'subdomain') and context['request'].subdomain is not None:
```

Comment:
I _think_ it's more Pythonic / duck-typing-style to just try to access the attribute rather than test with hasattr. So I might write this as:

```
try:
    subdomain = context['request'].subdomain
except AttributeError:
    return output
```

... and then use `subdomain` rather than `context['request'].subdomain` in the rest.